server/ansa/macros/translate_text.py

```diff
@@ -0,0 +1,35 @@
+import json
+import requests
+from flask import current_app as app
+from .process_html import process_html
+
+"""
+    Translate text from body_html
+"""
+
+
+def translate(text=''):
+    URL_TRANSLATION = app.config["ANSA_TRANSLATION_URL"]
+    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+    data = {'text': text, 'lang': 'en', 'target': 'it'}
```

Comment:
they might also translate the other way around, would make sense to translate to it if `item.language` is en and vice versa